src/mole_C++/utils.cpp

```diff
@@ -149,6 +192,15 @@ sp_mat Utils::spjoin_cols(const sp_mat &A, const sp_mat &B) {
   return result;
 }
 
+/**
+ * @brief An analog to the MATLAB 2D meshgrid operation
```

Comment:
That doesn't explain what it does. 
We cannot assume everybody is familiar with Matlab.

Reply:
Beter explanations are given in #[commit 0295c8f](https://github.com/csrc-sdsu/mole/commit/0295c8fdbeebed3a51260cb1276d185a8270844d#diff-5b1616b1306aa614c3da82d8556ffbcdc3b8e51a7f3d5f32007f141992d3b8bdR68-R74)

Reply:
I left the brief, but included a longer explanation.

https://github.com/csrc-sdsu/mole/blob/0295c8fdbeebed3a51260cb1276d185a8270844d/src/mole_C%2B%2B/utils.h#L65-L73

https://github.com/csrc-sdsu/mole/blob/0295c8fdbeebed3a51260cb1276d185a8270844d/src/mole_C%2B%2B/utils.h#L83-L89